doc/dev/mds_internals/quiesce.rst

```diff
@@ -0,0 +1,86 @@
+MDS Quiesce Protocol
+====================
+
+The MDS quiesce protocol is a a mechanism for "quiescing" (quieting) a
+subvolume in the file system, stopping all write (and most read) I/O.  A
+subvolume is a subtree in the file system marked with the "ceph.dir.subvolume"
+flag (like the mgr volumes plugin subvolumes).
+
+The purpose of this API is to prevent multiple clients from interleaving reads
+and writes across an eventually consistent snapshot barrier where out-of-band
+communication exists between clients. This communication can lead to clients
+wrongly believing they've reached a checkpoint that is mutually recoverable to
```

Comment:
s/to//

Reply:
I would think this is correct, as it's recovering **to** a snapshot?

Reply:
This is a nice summary of the quiesce feature as the means to achieve consistent snapshots!
Let's have a TODO to link to the `subvolume quiesce` command.